ow_plexil/src/plans/FaultHandlingPattern5.plp

```diff
@@ -0,0 +1,104 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Pattern summary: Execute a sequence of actions. If a fault is injected
+// during a related action, the action will stop and end in the ABORTED state. 
+// As long as the action is stopped, a backup action will repeat. If/when the 
+// fault is cleared, the original action will automatically be attempted again.
+
+// More specifically: If no faults are occurring, this plan will pan 180 
+// degrees, and then unstow and stow the arm.
+// If a relevant fault is injected, the plan will stop the current action 
+// and start repeating a panning action as backup to the original action.
```

Comment:
This clarifies a bit on my previous comment, but I'm still not sure I fully understand what a backup action is. Is it the same action as the one that was stopped?

Reply:
It is an action that is specifically isolated from the action that was just halted due to an injected fault. The idea is that if something happens to the arm, hence halting its action, we want another part of the lander to start doing something helpful that is not affected by arm faults. Hence, in this plan the backup action is panning the antenna.